scene/3d/xr_nodes.cpp

```diff
@@ -295,74 +290,70 @@ StringName XRNode3D::get_pose_name() const {
 }
 
 bool XRNode3D::get_is_active() const {
-	if (tracker.is_null()) {
+	if (tracker.is_null() || !tracker->has_pose(pose_name)) {
 		return false;
-	} else if (!tracker->has_pose(pose_name)) {
-		return false;
-	} else {
-		return true;
 	}
+	return true;
 }
 
 bool XRNode3D::get_has_tracking_data() const {
-	if (tracker.is_null()) {
-		return false;
-	} else if (!tracker->has_pose(pose_name)) {
-		return false;
-	} else {
+	if (get_is_active()) {
 		return tracker->get_pose(pose_name)->get_has_tracking_data();
 	}
+	return false;
 }
 
-void XRNode3D::trigger_haptic_pulse(const String &p_action_name, double p_frequency, double p_amplitude, double p_duration_sec, double p_delay_sec) {
+void XRNode3D::trigger_haptic_pulse(const String &p_action_name, double p_frequency, double p_amplitude, double p_duration_sec, double p_delay_sec) const {
```

Comment:
Not entirely sure if it makes sense to make this const. Haptics do change internal state even if this is offloaded to the XR interface, unless it's `trigger_haptic_pulse` is also const (which it shouldn't be)